lib/screens/health/health_screen.dart

```diff
@@ -36,28 +35,34 @@ class _HealthScreenState extends State<HealthScreen> {
   }
 
   Widget _content(HealthScreenViewModel viewModel) {
-    if (viewModel.showProjectEmptyScreen) {
+    if (viewModel.showLoadingScreen) {
+      return Scaffold(body:
+      Center(child:
+        Column(
+            mainAxisAlignment : MainAxisAlignment.center,
+            crossAxisAlignment : CrossAxisAlignment.center,
+            children: [
+              CircularProgressIndicator(value: viewModel.loadingProgress),
+              Container(
+                  padding: EdgeInsets.only(top: 16, left: 22, right: 22),
```

Comment:
We can use `EdgeInsets.symetric(vertical: 16, horizontal: 22)` if we would also like to have some inset at the bottom.